MWAppAuthPlugin/MWAppAuthPlugin/AppAuthStep/MWAppAuthStepViewController.swift

```diff
@@ -231,8 +231,17 @@ private extension MWAppAuthStepViewController {
             DispatchQueue.main.async {
                 self?.hideLoading()
                 switch response {
-                case .success:
-                    self?.goForward()
+                case .success(let credential):
+                    print(response)
```

Comment:
Shall we remove the `print` here?

Reply:
Sorry! I printed too much today 😅 Removing it!